app/app/urls.py

```diff
@@ -202,6 +202,7 @@
     url(r'^_administration/email/bounty_feedback$', retail.emails.bounty_feedback, name='bounty_feedback'),
     url(r'^_administration/email/start_work_expire_warning$', retail.emails.start_work_expire_warning, name='start_work_expire_warning'),
     url(r'^_administration/email/start_work_expired$', retail.emails.start_work_expired, name='start_work_expired'),
+    url(r'^_administration/email/gdpr_reconsent$', retail.emails.gdpr_reconsent, name='gdpr_reconsent'),
```

Comment:
We'll eventually need to update every `url()` to the new methods: `re_path()` and `path()` - It'll be easier if we use those for new urls being added.  Can we update this to `re_path()` ?